### DISCUSSION TITLE
Is it possible to edit the locations table?

### DISCUSSION THREAD

**Discussion by Spsrky** [code4recovery/12-step-meeting-list#749] [Q&A]
When an existing meeting already has a location assigned, editing the location address is not save - it reverts back to the original. Is there a work around for this or can the locations table be accessed to edit? Thanks!

**Comment by joshreisner** [comment]
we don't have a "locations table" in the database per se, we use wordpress's database schema of wp_posts, wp_postmeta, etc

but to your point, you need the ability to edit location information! sounds like you may be experiencing some type of bug. i'm able to edit a meeting's address and have that information save in our demo site, so sounds like there must be something different going on in your case.

first to eliminate one issue: when you enter an address that already exists in the system, it will try to load that location's name and notes, so you don't need to re-enter it. are you entering a meeting address that's already in use by other meetings?

second: when you edit the new address, does the map update? sometimes you need to click out of the address field and wait a beat for it to have a chance to do that, rather than immediately clicking the "Update" button. you'll know it's ready when the map updates to the new location.

let me know and we can take it from there thanks
